test/BCFKeyValueData.cc

```diff
@@ -329,8 +329,7 @@ TEST_CASE("BCFKeyValueData BCF retrieval") {
 
         // empty results
         s = data->dataset_bcf("NA12878D", hdr.get(), range(0, 0, 1000), records);
-        REQUIRE(s.ok());
-        REQUIRE(records.size() == 0);
+        REQUIRE((s == StatusCode::NOT_FOUND || records.size() == 0));
```

Comment:
It is not deterministic, based on bucket size choices. We might have empty buckets, due to the different choices of interval lengths.